sdcm/nemesis.py

```diff
@@ -3533,33 +3539,35 @@ def decommission_nodes(self, add_nodes_number, rack, is_seed: Optional[Union[boo
                           f'{str(exc)}').publish()
 
     def disrupt_grow_shrink_cluster(self):
-        self._grow_cluster(rack=0)
-        self._shrink_cluster(rack=0)
+        self._grow_cluster(rack=None)
+        self._shrink_cluster(rack=None)
 
     def disrupt_grow_shrink_new_rack(self):
+        if not self._is_it_on_kubernetes():
+            raise UnsupportedNemesis("Adding new rack is not supported for non-k8s Scylla clusters")
```

Comment:
Remove it. It is false statement and unneeded skip of nemesis for K8S.

The `racks` feature is supported in K8S. It is not tied to the real AZs anyhow. Because we can create several "racks" having them in one AZ and idea under it may be to use different node configurations (i.e. asymmetric cluster). Only affinity rules and node selectors, when specified in a K8S `rack` configuration, imply some of AZs to be picked up.

Reply:
`if not self._is_it_on_kubernetes()` - for k8s it is not skipped.

Reply:
You are right, I miss-looked.